### PR TITLE
add product design in html css

### PR DESCRIPTION
**Pull request by nesmamanasra**
**In this, these were added:**
- Adding a product filling form
- And add the required formats
   Images:
add product design

### CODE REVIEW

File: addproduct.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This app is platform where you can sell or buy stuff">
+    <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">  
+    <title>GITShop</title>
+    <link rel="stylesheet" type="text/css" href="styles/product.css">   
+    <link rel="stylesheet" type="text/css" href="styles/home.css">
```
Comment (saharAdem) [inline addproduct.html]: there is no need to link your style to home style, remove this.

**Review by saharAdem — CHANGES_REQUESTED** [review]

File: addproduct.html (hunk 1 above)
Comment (nesmamanasra) [reply addproduct.html]: yes i did

File: styles/product.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    border:#E0995E;
+    background: #E0995E;
+    text-align: center;
+    padding: 15px;
+    border-radius: 30px;
+    color: #fff;
+    cursor: pointer;
+    text-transform: uppercase;
+    margin-bottom: 20px;
+}
```
Comment (bayanseder) [inline styles/product.css:101]: the  2 button should have the same width

**Comment by bayanseder** [comment]
don't forget to make it responsive

**Review by bayanseder — APPROVED** [review]

**Pull request merged**